src/main/java/co/novu/api/tenants/TenantsApi.java

```diff
@@ -0,0 +1,35 @@
+package co.novu.api.tenants;
+
+import co.novu.api.tenants.requests.TenantRequest;
+import co.novu.api.tenants.responses.BulkTenantResponse;
+import co.novu.api.tenants.responses.DeleteTenantResponse;
+import co.novu.api.tenants.responses.TenantResponse;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.PATCH;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+public interface TenantsApi {
+
+    String ENDPOINT = "tenants";
+
+    @GET(ENDPOINT)
+    Call<BulkTenantResponse> getTenants(@Query("page") Integer page, @Query("limit") Integer limit);
```

Comment:
You can use @QueryMap for this instead. You can have a look at the other PRs for guidance

Reply:
@mayorJAY 
QueryMap do not allow to pass null values.
If we want to have `QueryMap` we should add some defaults in case page and limit is not specified...

Reply:
Okay, I put the limit and page to map only if they are not null. That can help as well 👍